Add Resource Drivers section  to extending Kubernetes documentation (#43742)
kubernetes/website: content/en/docs/reference/glossary/ResourceClaimParameters.md

---
title: ResourceClaimParameters
id: ResourceClaimParameters
date: 2023-10-16
full_link: /docs/concepts/scheduling-eviction/dynamic-resource-allocation/
short_description: >
  Specification of what and how much of resources the ResourceClaim needs.
aka:
tags:
- extension
---

>  {{< glossary_tooltip term_id="resource-driver" text="Resource Driver">}}-specific object, subject

Reviewer: ```suggestion
 Parameters for a{{< glossary_tooltip term_id="resource-driver" text="resource driver">}}, and subject
```

> to vendor implementation. Optional. Typically contains quantity and characteristics of the requested

Reviewer: Most objects are optional; for example, you don't need to have a Deployment. I suggest we link to the page about DRA for readers to learn more details.

resources.

<!--more-->

> Not part of core Kubernetes. Referenced in `ParametersRef` field of

Reviewer: ```suggestion
```
Instead, I suggest we link to the page about DRA for readers to learn more details.
